src/schema/bookmarks.ts

```diff
@@ -555,12 +555,11 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         info,
         (builder) => ({
           ...builder,
-          queryBuilder: builder.queryBuilder.where(
-            `"${builder.alias}"."userId" = :userId`,
-            {
+          queryBuilder: builder.queryBuilder
+            .where(`"${builder.alias}"."userId" = :userId`, {
               userId: ctx.userId,
-            },
-          ),
+            })
+            .addOrderBy(`LOWER("${builder.alias}"."name")`, 'ASC'),
```

Comment:
we probably need order by index with lower here

Reply:
can you also add test for this

Reply:
Seems that [we cannot add lower to index](https://github.com/typeorm/typeorm/issues/327), but maybe this will not be too heavy as query because we have only few folders per user, not?

I'm going to add the test case still ✔️

Reply:
@ilasw you can add the index manually through migration and say `syncronize: false`, we have some examples already

Reply:
Yeah we have some indeed:
https://github.com/dailydotdev/daily-api/blob/98947bf22e7ccd227bd5d86110f21744346ab4b3/src/migration/1707983477045-NewIndexes.ts#L16-L18

Reply:
Thanks for suggestion, added ✅

Reply:
Made it a bit more specific and `@Index` on entity. Good to go!